src/Globe/BuilderEllipsoidTile.js

```diff
@@ -120,14 +120,14 @@ BuilderEllipsoidTile.prototype.OBB = function OBBFn(params) {
     }
 
     maxHeight *= 0.5;
-    var width = Math.abs(maxV.z - minV.z) * 0.5;
+    var width = Math.abs(maxV.y - minV.y) * 0.5;
     var height = Math.abs(maxV.x - minV.x) * 0.5;
     var delta = height - Math.abs(cardin3DPlane[5].x);
     var max = new THREE.Vector3(width, height, maxHeight);
```

Comment:
Fixed. The problem came from the fact that the default up vector was (0,1,0) instead of (0,0,1). I changed it at the initialisation of the scene, in c3DEngine.